python/py-sure/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-sure
+version             1.4.11
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
+maintainers         nomaintainer
+
+description         Utility belt for automated testing in python for python
+long_description    ${description}
+
+homepage            http://github.com/gabrielfalcao/sure
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  3c8d5271fb18e2c69e2613af1ad400d8df090f1456081635bd3171847303cdaa \
+                    rmd160  fb6c37da7c2d005c6874d266fbc8d2edcb21d833 \
+                    size    45933
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
```

Comment:
you'll need to add (see https://github.com/gabrielfalcao/sure/blob/fa5c87c495b7237f6e6920bc9acee2e05082f681/setup.py#L61):
```
depends_build-append \
                    port:py${python.version}-mock \
                    port:py${python.version}-six
```

Reply:
I would say that the `setup.py` is lying. I only see those libraries used in tests.

I do agree that I miss all three (`mock`, `nose`, `six`) in test dependencies, but I fail to see why thes two would be build dependencies.

Reply:
I would agree that they are probably not build-depdencies, but that's what upstream says ;) 

For sure both of them are runtime dependencies as they are imported in several python files in the package. So sure, if you want you can add them to ```depends_run``` instead of ```depends_lib```.

Reply:
I added them to `depends_run`. If they were defined as build dependencies as you initially suggested, they could be removed by `port reclaim`.

Reply:
oh, sorry... that was not at all what I intended to suggest - I meant to write ```depends_lib``` ;) But, yes ```depends_run``` is even more accurate.